packages/flame/lib/src/collisions/broadphase/quadtree/quad_tree_broadphase.dart

```diff
@@ -35,8 +34,8 @@ class QuadTreeBroadphase<T extends Hitbox<T>> extends Broadphase<T> {
 
   final activeCollisions = HashSet<T>();
 
-  ExternalBroadphaseCheck broadphaseCheck;
-  ExternalMinDistanceCheck minimumDistanceCheck;
+  final ExternalBroadphaseCheck broadphaseCheck;
+  final ExternalMinDistanceCheck minimumDistanceCheck;
```

Comment:
We could let the users change this during runtime, or do you see any downside to that?

Reply:
I missed the purpose of this. Reverting to the previous.